pkg/core/interop/contract/call.go

```diff
@@ -70,11 +70,23 @@ func Call(ic *interop.Context) error {
 	}
 	hasReturn := md.ReturnType != smartcontract.VoidType
 
-	ic.InvocationCalls = append(ic.InvocationCalls, state.ContractInvocation{
-		Hash:      u,
-		Method:    method,
-		Arguments: stackitem.NewArray(args),
-	})
+	if ic.Chain.GetConfig().Ledger.SaveInvocations {
```

Comment:
`GetConfig()` for every call is too costly, we need some new field in `ic` inited on `NewContext()` (just like `Network` and `Hardforks` are initialized currently).